Author: Niko Tyni <ntyni@debian.org>
Subject: Help apxs find the correct directory
 As a comment inside apxs2 indicates:
 # Debian doesn't have a CFG_PREFIX [...]
 so use SYSCONFDIR instead to shut up apxs warnings.
Forwarded: not-needed
Last-Update: 2011-03-22
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- a/Apache-Test/lib/Apache/TestConfigParse.pm
+++ b/Apache-Test/lib/Apache/TestConfigParse.pm
@@ -87,7 +87,7 @@
                        'user-supplied $base' ],
                  [ $self->{inherit_config}->{ServerRoot},
                        'httpd.conf inherited ServerRoot' ],
-                 [ $self->apxs('PREFIX'),
+                 [ $self->apxs('SYSCONFDIR'),
                        'apxs-derived ServerRoot' ]);
 
     # remove surrounding quotes if any
